Wattpad/kamu
# [PI-2384] update outdated deps

This PR fixes just enough stuff to (hopefully) unblock deploys for this repo. The major changes are the following:
- Bump node version to v16: This is needed for deploys to work
- Bump `sharp` library to v0.32.1: This makes sharp compatible with node v16
- Add `gulp@3.9.1` as a dev dependency: Allows devs to run tests locally
- Updated tests due to changes in #20 
- Delete the committed `node_modules` directory: Why was this ever committed?

## Testing

1. Clone repo locally
2. Switch to node v16.15.0 (easiest to do by using nvm)
3. Run `npm install`
4. Run `npm run test`
5. Ensure that tests pass ✅
